etc/config/c++.amazon.properties

```diff
@@ -392,6 +392,36 @@ compiler.clang_reflection.semver=(reflection)
 compiler.clang_reflection.options=-std=c++20 -freflection-ts -stdlib=libc++
 compiler.clang_reflection.notification=Experimental Reflection Support; see <a href="https://github.com/cplusplus/reflection-ts" target="_blank" rel="noopener noreferrer">github<sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a>
 
+group.llvm-amdgpu.compilers=llvm-amdgpu-trunk:llvm-amdgpu-40502:llvm-amdgpu-50002:llvm-amdgpu-50103:llvm-amdgpu-50203
```

Comment:
Ditto here: does `clang-amdgpu-` work? that is more descriptive as an ID but the actual project we're building is `clang-rocm-*`:

```
Sep/21 14:46 admin-node~/infra (main|✔) $ ls -ld /opt/compiler-explorer/clang-rocm-*
drwxr-xr-x 8 root root 6144 Sep 21 14:30 /opt/compiler-explorer/clang-rocm-4.5.2
drwxr-xr-x 8 root root 6144 Sep 21 14:34 /opt/compiler-explorer/clang-rocm-5.0.2
drwxr-xr-x 8 root root 6144 Sep 21 14:38 /opt/compiler-explorer/clang-rocm-5.1.3
drwxr-xr-x 8 root root 6144 Sep 21 14:42 /opt/compiler-explorer/clang-rocm-5.2.3
lrwxrwxrwx 1 root root   25 Sep 21 14:19 /opt/compiler-explorer/clang-rocm-trunk -> clang-rocm-trunk-20220921
drwxr-xr-x 8 root root 6144 Sep 21 14:17 /opt/compiler-explorer/clang-rocm-trunk-20220921
```

are the paths.

Reply:
`clang-rocm` is probably fine. The compiler is installed via the `rocm-llvm` package, and it describes itself as:
```
AMD clang version 14.0.0 (https://github.com/RadeonOpenCompute/llvm-project roc-5.2.3 22324 d6c88e5a78066d5d7a1e8db6c5e3e9884c6ad10e)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /opt/rocm-5.2.3/llvm/bin
```